Content.Server/Sticky/Systems/StickySystem.cs

```diff
@@ -20,15 +20,51 @@ public sealed class StickySystem : EntitySystem
     [Dependency] private readonly SharedHandsSystem _handsSystem = default!;
     [Dependency] private readonly SharedInteractionSystem _interactionSystem = default!;
     [Dependency] private readonly SharedAppearanceSystem _appearance = default!;
+    [Dependency] private readonly EntityLookupSystem _lookup = default!;
+    private ISawmill _sawmill = default!;
 
     private const string StickerSlotId = "stickers_container";
 
     public override void Initialize()
     {
         base.Initialize();
+        SubscribeLocalEvent<TryStickOnSpawnComponent, MapInitEvent>(OnAutoStickMapInit);
+
         SubscribeLocalEvent<StickyComponent, StickyDoAfterEvent>(OnStickFinished);
         SubscribeLocalEvent<StickyComponent, AfterInteractEvent>(OnAfterInteract);
         SubscribeLocalEvent<StickyComponent, GetVerbsEvent<Verb>>(AddUnstickVerb);
+
+        _sawmill = Logger.GetSawmill("sticky");
```

Comment:
Sawmills on systems are deprecated, use the Log property as it gets its own sawmill.